openedx_lti_tool_plugin/http.py

```diff
@@ -0,0 +1,20 @@
+"""HTTP objects for openedx_lti_tool_plugin."""
+import logging
+
+from django.http import HttpResponseBadRequest
+
+log = logging.getLogger(__name__)
+
+
+class LoggedHttpResponseBadRequest(HttpResponseBadRequest):
+    """A HTTP 400 response class that sends the response to the log."""
+
+    def __init__(self, *args, **kwargs):
+        """HTTP response __init__ method.
+
+        Args:
+            *args: Variable length argument list.
+            **kwargs: Arbitrary keyword arguments.
+        """
+        super().__init__(*args, **kwargs)
+        log.error(args[0])
```

Comment:
Is it necessary to check the existence of index "0"? @kuipumu

Reply:
@Squirrel18 I will modify it to be called inside a try except block instead